Adyen/UI/Form/Items/FormImageItem/FormImageItem.swift

```diff
@@ -0,0 +1,71 @@
+//
+// Copyright (c) 2022 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import UIKit
+
+/// Simple form item that represent a single UImage element.
+@_spi(AdyenInternal)
+public class FormImageItem: FormItem, Hidable {
+
+    public var isHidden: AdyenObservable<Bool> = AdyenObservable(false)
+
+    public var subitems: [FormItem] = []
+
+    /// The name of the Image.
+    public var name: String
+
+    public var identifier: String?
+
+    /// The size of the Image.
+    public var size: CGSize?
```

Comment:
Maybe make default size instead of "magic numbers" in `renderImage` ?

```suggestion
    public var size: CGSize = .init(width: 46, height: 46)
```

Reply:
Fixed. Here is the commit: 4626cebdcb910d5b2838ccd4b15c3d49665b6bda